docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -76,15 +80,28 @@ $ tctl create -f access.yaml
 
 #### Export access-plugin Certificate
 
-Teleport Plugin use the `access-plugin` role and user to perform the approval. We export the identity files, using [`tctl auth sign`](https://gravitational.com/teleport/docs/cli-docs/#tctl-auth-sign).
-
-```bash
-$ tctl auth sign --format=tls --user=access-plugin --out=auth --ttl=720h
-# ...
-```
-
-The above sequence should result in three PEM encoded files being generated: auth.crt, auth.key, and auth.cas (certificate, private key, and CA certs respectively).  We'll reference the auth.crt, auth.key, and auth.cas files later when [configuring the plugins](#configuring-teleport-slack).
+Teleport Plugin use the `access-plugin` role and user to perform the approval. We export the identity files, using [`tctl auth sign`](https://goteleport.com/docs/cli-docs/#tctl-auth-sign).
 
+<Tabs>
+<TabItem label="Teleport Cloud or via Proxy connection">
+  If you are using Teleport Cloud or want to have the plugin only connect through the proxy ports you must use a identity file.  When you `tctl auth sign` through Teleport Cloud or remotely you must have [impersonation](https://goteleport.com/docs/access-controls/guides/impersonation/) rights on your user's role.
+
+  ```bash  
+  $ tctl auth sign --user=access-plugin --format=file --out=access-plugin.pem --ttl=720h
```

Comment:
I was able to connect with: `sudo tctl auth --auth-server=slacktest.go-teleport.com:3025 sign --format=tls --user=access-plugin --out=auth --ttl=1h`

Reply:
yes, that will work when you can access the auth server